designsafe/apps/projects/models/agave/experimental.py

```diff
@@ -62,19 +62,44 @@ class Experiment(RelatedEntity):
     project = fields.RelatedObjectField(ExperimentalProject)
     dois = fields.ListField('Dois')
 
-    def to_datacite_json(self):
+    def to_datacite_json(self, project={}):
         """Serialize object to datacite JSON."""
         attributes = super(Experiment, self).to_datacite_json()
-        attributes["subjects"] = attributes.get("subjects", []) + [
-            {"subject": self.experimental_facility.title(), }
+        if hasattr(self, 'facility') and len(self.experimental_facility) and ('None' not in self.experimental_facility):
```

Comment:
Can experiments have a `facility` attribute? I think this expression might always evaluate to false.

Reply:
good point. It should be 'experimental_facility'